core/src/tasks/profile/destroyEvents.ts

```diff
@@ -9,19 +9,18 @@ export class ProfileDestroyEvents extends CLSTask {
     this.frequency = 0;
     this.queue = "events";
     this.inputs = {
-      id: { required: true },
```

Comment:
is there anything we need to put in place to transition here?

Reply:
For example, there are things queue'd up with the old inputs.

Reply:
I think this is safe - There's no automated task that will destroy a profile besides merging them - from events.  It's rather unlikely folks will have one of these tasks queued up when the upgrade.